src/components/Sing/SequencerNote.vue

```diff
@@ -93,6 +116,12 @@ const props = defineProps<{
   /** ノートが重なっているか */
   isOverlapping: boolean;
   previewLyric: string | null;
+  /** ノートがリサイズ中か */
+  isResizingNote: boolean;
+  /** ノートが右方向にリサイズ中か */
+  isResizingRight: boolean;
+  /** ノートが左方向にリサイズ中か */
+  isResizingLeft: boolean;
```

Comment:
（あまり重要度の高くないコメントです）

これたぶんこんな感じにすると１変数にできたりします。そのままでも問題なさそう！
```suggestion
  /** ノートのリサイズ状態。undefinedならリサイズ中ではない、right・leftなら右・左方向にリサイズ中 */
  noteResizing: undefined | "right" | "left";
```